pom.xml

```diff
@@ -125,6 +125,19 @@
             <artifactId>undertow-websockets-jsr</artifactId>
             <version>${undertow.version}</version>
         </dependency>
+        
+        <!-- RESTEasy Undertow Integration -->
```

Comment:
add digest rules to the enforcer: https://github.com/iotaledger/iri/blob/9821a60f8fea0aa443f13b3158208f8c4c813908/pom.xml#L369